src/test/java/com/kainos/ea/backend/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,94 @@
+package com.kainos.ea.backend.controllers;
+
+import com.kainos.ea.backend.models.User;
+import com.kainos.ea.backend.services.UserService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+public class UserControllerTest {
+
+    @Mock
+    private UserService userService;
+    @Mock
+    private BCryptPasswordEncoder passwordEncoder;
+
+    @Test
+    public void when_verifyUserCalledWithValidUser_expect_true() {
+        User validUser = new User("mail@email.com", "strong_password");
+        Mockito.when(userService.validateUser(validUser)).thenReturn(true);
+        Mockito.when(userService.doCredentialsMatch("mail@email.com", "strong_password", passwordEncoder)).thenReturn(true);
+        UserController userController = new UserController(userService, passwordEncoder);
+        Boolean results = userController.verifyUser(validUser);
+        Mockito.verify(userService).validateUser(validUser);
+        assertTrue(results);
+    }
+
+    @Test
+    public void when_verifyUserCalledWithUnknownUser_expect_false() {
+        User invalidUser = new User("mail@email.com", "strong_password");
+        Mockito.when(userService.validateUser(invalidUser)).thenReturn(true);
+        Mockito.when(userService.doCredentialsMatch("mail@email.com", "strong_password", passwordEncoder)).thenReturn(false);
```

Comment:
All the other tests have whitespace between the arrange, act, assert steps. I think it would be nicer to be consistent and improve readability

Reply:
Whitespaces are back